src/commands.ts

```diff
@@ -724,11 +732,11 @@ export function viewMir(ctx: Ctx): Cmd {
 
 export function interpretFunction(ctx: Ctx): Cmd {
   return async () => {
-    const { document, position } = await workspace.getCurrentState();
+    const {document, position} = await workspace.getCurrentState();
     if (!isRustDocument(document)) return;
```

Comment:
formatting issue too

Reply:
I think it would be a good idea to add local coc-server format configuration.